CMakeLists.txt

```diff
@@ -159,16 +119,9 @@ else()
     message(STATUS "Setting logging level to '${LOGGING_LEVEL}'")
 endif(NOT LOGGING_LEVEL)
 
-target_compile_definitions(rmm PUBLIC SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${LOGGING_LEVEL})
-
-target_link_libraries(rmm PUBLIC spdlog::spdlog_header_only)
-
```

Comment:
This should remain -- when using `rmm`, the include paths for spdlog need to be made available.

Reply:
Done.